docs/packaging/submitting-a-package.md

```diff
@@ -63,45 +94,78 @@ For files that must be removed, you must do so using git: `git rm someFile`
 
 Likewise, for renaming a file, you must do so via git: `git mv someFile someFileName2`
 
-### Commit
+### Commit your Changes
+
+**Check your branch**
+
+Get the status of the branch with `git status`. Make sure all the files you changed are staged, and that there are no untracked files. The git status should say your branch is clean.
+
+Now that you've tested and reviewed your change, when you're happy with it, it's time to commit your changes with `git commit`.
+
+Make sure you provide a meaningful summary (with the package name) and a separate body to your commit message. Package update details should come from the upstream project release notes, and contain no links. There may also be a section for Solus specific work.
+
+There should be a summary line, a blank line, and then the rest of the commit message. Bullet point lists should start with a dash.
+
+Here is an example in our standard format:
 
-Once you're happy with your change, and you have verified locally that it works by having first built and
-installed it, it's time to commit your changes with `git commit`.
+```
+Update foo to 1.2.3
+
+**Changelog**
+
+Bugfixes:
+
+- Fixed a crash
+- Something else
+
+Enhancements:
+
+- Implemented a feature
+- Error when encountering a thing
+
+**Full release notes:**
+- [1.2.3](https://github.com/foo/foo/releases/tag/v1.2.3)
+
+## Test Plan
+
+- Launched the application
+- Exercised the UI
+- Exercised some feature
+```
 
-Make sure you provide a meaningful summary and a separate body to your commit message. For more information
-on suitable commit messages, please check the [tooling central documentation](https://github.com/solus-project/tooling-central/blob/master/README.rst#using-git).
+For more information on suitable commit messages, please check the [tooling central documentation](https://github.com/solus-project/tooling-central/blob/master/README.rst#using-git).
 
-- If you want to link this patch to an issue on the Dev Tracker, simply mention it in your commit message: `The inclusion of <somepackage> fixes #1234`
+- If you want to link this pull request to an existing issue, simply mention it in your commit message: `The inclusion of <somepackage> fixes #1234`
 - If you need a change to depend on another change, mention it in the commit message too: `Depends on #123`
```

Comment:
Since issues are in a different repository (and organization), it's best to put in the full URL. You can probably get away with just `getsolus/packages#1234` (or something like that; I don't quite remember the correct regex off the top of my head), but just copying the full URL is likely faster and easier.

Reply:
Good point. Fixed.